src/robusta/integrations/webex/sender.py

```diff
@@ -0,0 +1,253 @@
+from ...core.reporting.base import Finding, FindingSeverity
+from ...core.reporting.utils import is_image, convert_svg_to_png, SVG_SUFFIX, PNG_SUFFIX
+from ...core.reporting.blocks import (
+    MarkdownBlock,
+    BaseBlock,
+    FileBlock,
+    TableBlock,
+    List,
+)
+from ...core.sinks.transformer import Transformer
+
+from PIL import Image
+from io import BytesIO
+from webexteamssdk import WebexTeamsAPI
+from fpdf import FPDF
+import os
+
+
+INVESTIGATE_ICON = "\U0001F50E"
+SILENCE_ICON = "\U0001F515"
+
+MAX_BLOCK_CHARS = 7439
+
+ADAPTIVE_CARD_VERSION = "1.2"
+ADAPTIVE_CARD_SCHEMA = "http://adaptivecards.io/schemas/adaptive-card.json"
+ATTACHMENT_CONTENT_TYPE = "application/vnd.microsoft.card.adaptive"
+
+CARD_TYPES = {
+    1: "AdaptiveCard",
```

Comment:
I think an enum could be more suitable instead of these dicts. you could use string enums as well

Reply:
Updated. Let me know if its ok or anything needs to be changed.